requirements-base.txt

```
@@ -251,7 +253,7 @@ patsy==0.5.3
     # via statsmodels
 pbr==5.11.1
     # via jira
-pdpyras==4.5.2
+pdpyras==5.0.1
```

```suggestion
pdpyras==4.5.2
```

5.0.1 introduces a bug that prevents retrieving on-call email, see:

https://github.com/PagerDuty/pdpyras/pull/103